src/components/book/book-item.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ author, title }) => (
+  <li>
+    {author}
+    {' '}
+    {title}
+  </li>
```

Comment:
✅ The Book component takes the book data as a prop and renders the title and the author 👍🏻